admin-dev/filemanager/include/php_image_magician.php

```diff
@@ -2539,15 +2539,6 @@ public function saveImage($savePath, $imageQuality="100")
   #             * bmp files have no native support for bmp files. We use a
   #       third party class to save as bmp.
     {
-
-    // *** Perform a check or two.
-    if (!is_resource($this->imageResized)) {
```

Comment:
Why not check both? It seems that `$this->imageResized` can be `false` so it would make sense to test it?
```php
if (!is_resource($this->imageResized) && !($this->imageResized instanceof GdImage)) {
```

Reply:
@FabienPapet I don't see where the check is done?

Reply:
We already check if the file exists in the constructor, but the constructor does not throw any exception if the file does not exist or if the extension is not supported.... 
Anyway, as it can be false, I've reverted and added the condition. 

I really hope we can get rid of this library soon